test/test_datapipe.py

```diff
@@ -567,7 +568,7 @@ def _return_self(x):
         with self.assertRaises(TypeError):
             len(batch_dp)
 
-    def test_flatmap_datapipe(self):
+    def test_flatmap_iterdatapipe(self):
```

Comment:
Maybe better for a different PR?

Reply:
It is small enough that I decided to keep it here.